### ISSUE TITLE
Missing require for helpers.php ?

### ISSUE THREAD

**Issue by Lordbufu** [laracasts/The-PHP-Practitioner-Full-Source-Code#3]
Hello,

Reviewing this after having completed the training that it belongs to, it seems the helper function (that have been split out of the bootstrap file) are not required any where in the code.

So when i was trying to clean up my own code i noticed my server not being able to find the view function for example.

Unless i missed something terrible obvious i think this file needs to be required, since composer dint help much so that is how i solved it for my self simply add:
`require 'core/helpers.php';`

to the entry point (index.php in the root of the website).
Hope this helps.

Kind regards
Marco/Bufu

**Comment by surgiie** [comment]
Did you run composer dump-autoload?

**Comment by Lordbufu** [comment]
Yes, the only way i got it to load so far via composer, is adding a line to the composer.json that requires that specific file. It might be Cloud9 being a bit grumpy, think i can finish my server at home this weekend and ill see if the problem is still there without Cloud9.
